src/routes/tasks/components/ErrorAlert.tsx

```diff
@@ -56,26 +56,6 @@ const ErrorAlert = ({ task }: { task: AnyTupleT }): JSX.Element | null => {
                 }
             />
         );
-    } else if (task.error_type === ErrorT.build) {
-        return (
-            <ErrorAlertBase
-                title="Build error"
-                description={
-                    <>
-                        An error occurred when building the container for the
-                        task execution. Please check your algorithm archive or
-                        contact{' '}
-                        <Link
-                            href="https://lfaifoundation.slack.com/#substra-help"
-                            isExternal
-                        >
-                            substra-help Slack channel
-                        </Link>{' '}
-                        to get access to logs.
-                    </>
-                }
-            />
-        );
     } else {
         // task.error_type === ErrorT.execution
```

Comment:
Can you just had to this comment that it also includes `ErrorT.build` now :)

Reply:
Oh, good point! Looking at it closer, the block following this line has multiple mentions of the error being an "execution" error, which is not accurate anymore since it might now be a "buid" error. 

Not sure how you want to address it from a UX perspective. Should we just duplicate the block and change "task execution" to "algo build", or something?

Reply:
Oh yes nice catch ! Discussed it with David, we can just duplicate the block and let this part of the description `An error occurred when building the container for the task execution.` with `Build Error` title. Anyway we have to rethink this block as the link to slack channel to get logs is not working now that this is open-source.

Reply:
done